### DISCUSSION TITLE
can whisper-timestamped do batch inference/transcription?

### DISCUSSION THREAD

**Discussion by afsara-ben** [linto-ai/whisper-timestamped#179] [General]
i am running `model.transcribe(audio_list)` but i get error `TypeError: expected np.ndarray (got list)`

**Comment by Jeronymous** [comment]
Can you give more elements to reproduce?
what is audio_list?

**Reply by afsara-ben** [reply]
`audio_list=['sample1.mp3','sample2.mp3']` is a list containing path to audio files

**Comment by Jeronymous** [comment]
Ah, of course! Thanks.

So, indeed batch computation is not implemented.

We should give a more understandable error message maybe...
